examples/edits.py

```diff
@@ -1,20 +1,34 @@
 import nextcord
 import asyncio
 
-class MyClient(nextcord.Client):
+from nextcord.ext import commands
+
+
+class Bot(commands.Bot):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
     async def on_ready(self):
```

Comment:
on_ready should be removed from all examples where that isn't the example like basic_bot or the one in the readme